force-app/main/default/lwc/productFilter/__tests__/productFilter.test.js

```diff
@@ -132,4 +132,52 @@ describe('c-product-filter', () => {
             );
         }
     });
+
+    describe('getPicklistValues @wire error', () => {
+        it('shows error message elements', () => {
+            const error = {
+                details: {
+                    body: {
+                        message: 'error from test',
+                    },
+                },
+            };
+            const element = createElement('c-product-filter', {
+                is: ProductFilter,
+            });
+            document.body.appendChild(element);
+            getPicklistValuesAdapter.error(error);
```

Comment:
The error here is assumed to be of the shape ` error.details.body.message`. I assume that's how all LDS wire adapters send down their error payloads. Should we have the test wire adapter that's returned from `registerLdsTestWireAdapter` accept a string to the `error()` function and automatically put it in that format?

Reply:
LDS follows UIAPI’s shape: https://developer.salesforce.com/docs/atlas.en-us.uiapi.meta/uiapi/. This differs from what you have so I need to double-check LDS source code. 

We should simplify the error() function as you suggest.